data/mods/Magiclysm/monsters/monsters.json

```diff
@@ -328,5 +328,100 @@
     "anger_triggers": [ "PLAYER_CLOSE" ],
     "death_function": [ "NORMAL" ],
     "flags": [ "SEES", "SMELLS", "HEARS", "STUMBLES", "PATH_AVOID_FIRE", "REGENERATES_1", "REVIVES" ]
+  },
+  {
+    "id": "mon_claygolem",
+    "type": "MONSTER",
+    "name": "clay golem",
+    "description": "Large human-like golem made from clay.  Its proportions are off and seems fragile.",
+    "default_faction": "robot",
+    "bodytype": "human",
+    "species": [ "ROBOT" ],
+    "diff": 2,
+    "volume": "120 L",
+    "weight": 122000,
+    "hp": 100,
+    "speed": 80,
+    "material": [ "clay" ],
+    "symbol": "X",
+    "color": "brown",
+    "aggression": 10,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 10,
+    "melee_cut": 5,
+    "armor_bash": 10,
+    "armor_cut": 10,
+    "dodge": 0,
+    "vision_day": 40,
+    "vision_night": 40,
+    "anger_triggers": [ "PLAYER_CLOSE", "HURT" ],
+    "death_function": [ "BROKEN" ],
+    "flags": [ "SEES", "NO_BREATHE", "ACID_IMMUNE", "LOUDMOVES" ]
+  },
+  {
+    "id": "mon_stonegolem",
+    "type": "MONSTER",
+    "name": "stone golem",
+    "description": "Large human-like golem made from stone.  Its fists look similar to rockets.",
+    "default_faction": "robot",
+    "bodytype": "human",
+    "species": [ "ROBOT" ],
+    "diff": 2,
+    "volume": "120 L",
+    "weight": 162000,
+    "hp": 250,
+    "speed": 60,
+    "material": [ "stone" ],
+    "symbol": "X",
+    "color": "gray",
+    "aggression": 10,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 3,
+    "melee_dice_sides": 8,
+    "melee_cut": 6,
+    "armor_bash": 20,
+    "armor_cut": 20,
+    "dodge": 0,
+    "vision_day": 40,
+    "vision_night": 40,
+    "special_attacks": [ { "type": "spell", "spell_id": "rocket_punch", "spell_level": 5, "cooldown": 10 } ],
+    "anger_triggers": [ "PLAYER_CLOSE", "HURT" ],
+    "death_function": [ "BROKEN" ],
+    "flags": [ "SEES", "NO_BREATHE", "ACID_IMMUNE", "LOUDMOVES" ]
+  },
+  {
+    "id": "mon_irongolem",
+    "type": "MONSTER",
+    "name": "iron golem",
+    "description": "Large human-like golem made from iron.  Sometimes, green gas is escaping its mouth.",
```

Comment:
I'd reword:
"A large, humanoid golem made from iron. Some sort of noxious gas seems to be seeping from its mouth."